src/pages/Message.tsx

```diff
@@ -0,0 +1,73 @@
+import MessageContent from "../components/Message/MessageContent";
+import MessageListItem from "../components/Message/MessageListItem";
+
+const TEMP_USER: User = {
+  role: "Regular",
+  isOnline: false,
+  posts: [],
+  likes: [],
+  comments: [],
+  followers: [],
+  following: [],
+  notifications: [],
+  messages: [],
+  _id: "675c637f7812b71a3cef02b5",
+  fullName: "하치와레",
+  email: "abc2@gmail.com",
+  createdAt: "2024-12-13T16:40:31.755Z",
+  updatedAt: "2024-12-16T03:29:53.507Z",
+  image:
+    "https://res.cloudinary.com/learnprogrammers/image/upload/v1734108092/user/6559338b-237f-43f1-bf6d-764b9e133cf4.jpg",
+  coverImage: "",
+};
+
+export default function Message() {
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+  };
+
+  return (
+    <>
+      <section className="flex flex-row">
+        <article className="flex flex-col justify-between w-[721px] ">
+          <div className="px-10">
+            <div className="flex items-center mt-[38px] mb-[29px]">
+              <img
+                src={TEMP_USER.image || "/logo.png"}
+                alt="프로필 이미지"
+                className="w-[30px] h-[30px] mr-3 mb-1 rounded-full"
+              />
+              <p className="text-lg dark:text-white">
+                닉네임{/* {user?.fullName} */}
+              </p>
+            </div>
+            <MessageContent
+              isOutgoingMessage={false}
+              message="안녕하세요~~~~ 반갑습니다아~~~~ ^_^"
+            />
+          </div>
+          <div className="border-t border-gray-c8 px-10 pb-8">
+            <form className="flex flex-col" onSubmit={handleSubmit}>
+              <textarea
+                className="w-full mt-5 p-3 border border-gray-c8 rounded-[15px] whitespace-pre-wrap focus:text-gray-22 placeholder:text-[#cecece]"
```

Comment:
```suggestion
                className="w-full mt-5 p-3 border border-gray-c8 rounded-[15px] dark:border-gray-c8/50 resize-none"
```
textarea는 줄바꿈이 알아서 적용되는 입력란이기 때문에 whitespace-pre-wrap이 필요 없습니다~
그리고 글자 색과 placecholder은 기본으로 #222로 적용되고 있어요! (globals.css에서 설정해뒀습니다)